plugin/tasks.py

```diff
@@ -26,7 +26,7 @@ def create_instance(config, **kwargs):
     gcp = GoogleCloudPlatform(config['auth'],
                               config['project'],
                               config['scope'])
-    response = gcp.create_instance(ctx.node.name,
+    response = gcp.create_instance(ctx.node.id,
```

Comment:
this should probably be ctx.instance.id to assure it is unique (both "ctx.node.name" and "ctx.node.id" are the same, and hold to the node name from the blueprint, which is something like "server", not something unique)